modules/coo-must-gather.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * observability/cluster_observability_operator/cluster-observability-operator-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="coo-must-gather_{context}"]
+= Using the must-gather tool
+You can use the must-gather tool to collect information about the Cluster Observability Operator resources.
+
+.Procedure
+. Navigate to the directory where you want to store the must-gather data.
+. Run the following command to collect cluster-wide must-gather resources:
++
+[source,terminal]
+----
+$ oc adm must-gather
+ --image-stream=openshift/must-gather \
+ --image=quay.io/redhat-user-workloads/cluster-observabilit-tenant/cluster-observability-operator-container
```

Comment:
Image name should be 
`registry.redhat.io/cluster-observability-operator/cluster-observability-rhel8-operator`